src/features/Articles/Article/Article.tsx

```diff
@@ -0,0 +1,61 @@
+import { ArticleAPI } from '@/api/Article';
+import Style from './Article.module.scss';
+import Image from 'next/image';
+import Link from 'next/link';
+
+export default function Article({
```

Comment:
Why do you need this component? Wouldn’t it be easier to remove it and put everything in the page?